taskcluster/fenix_taskgraph/transforms/build.py

```diff
@@ -92,10 +93,10 @@ def add_release_version(config, tasks):
     for task in tasks:
         if task.pop("include-release-version", False):
             # TODO Move GIT_TAG to a parameter
-            git_tag = os.environ['GIT_TAG']
+            git_tag = os.environ.get('GIT_TAG', 'v0')
```

Comment:
Hmm, I wonder if `v0` will be confused with a real tag, perhaps we should make the default into something that is more clearly a "not-set" default?

Reply:
Good call. Done.